org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/general/completion/FilePathCompletionParticipant.java

```diff
@@ -89,17 +89,19 @@ public void onDTDSystemId(String value, ICompletionRequest request, ICompletionR
 
 	private static void addCompletionItems(String value, ICompletionRequest request, ICompletionResponse response)
 			throws Exception {
-		String fullValue = value;
-		if (isEmpty(fullValue)) {
-			return;
-		}
 
 		DOMDocument xmlDocument = request.getXMLDocument();
 		String text = xmlDocument.getText();
-
 		// Get value and range for file path declared inside the attribute value
 		// ex value="file:///C:/fold|er"
 		int valuePathStartOffset = xmlDocument.offsetAt(request.getReplaceRange().getStart());
+
+		String fullValue = value;
+		boolean inDoctype = isInDoctype(valuePathStartOffset, xmlDocument);

```

Comment:
You don't need to have  `isInDoctype` method because `addCompletionItems` by `onDTDSystemId`, so you can add a boolean parameter `isInDoctype` in  `addCompletionItems` which is filled correctly.

Reply:
Good catch! I've fixed this